app/models/form/sales/questions/buyer2_working_situation.rb

```diff
@@ -27,7 +27,6 @@ def initialize(id, hsh, page)
     "0" => { "value" => "Other" },
     "10" => { "value" => "Buyer prefers not to say" },
     "7" => { "value" => "Full-time student" },
-    "9" => { "value" => "Child under 16" },
```

Comment:
was this intentional?

Reply:
Yup! We don't allow this combination anyway, cause buyer 2 age can't be under 16 and it would trigger a validation if this option is selected in conjunction with age over 16. This also makes it behave the same way as buyer 1 question